captainhook.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
**New file**

Added to
- run regex checks on commit message
- run git secrets and unit tests as part of pre-commit